src/System.Memory/src/System/Span.cs

```diff
@@ -386,6 +382,25 @@ public bool TryCopyTo(Span<T> destination)
             return false;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private unsafe void CopyBlocks(ref T dst, ref T src, int length)
```

Comment:
This is slow span only and this implementation is already 9x faster than what we had. I could add that optimization if we really want it.